### DISCUSSION TITLE
Sort by any parameter in graphiql

### DISCUSSION THREAD

**Discussion by Leonbusy123** [hasura/graphql-engine#10521] [Question]
How to write graphiql in hasura if you want any field to be used as a parameter to complete forward/reverse order or filtering

**Comment by meetzaveri** [comment]
Hey @Leonbusy123 

Can you please give us more inputs like providing a query and expected query you want ? I believe you must have gone through this doc section of [Sort query results](https://hasura.io/docs/2.0/queries/postgres/sorting/) but if not please check it out.

In terms of if you find anything lacking in GraphiQL editor, you may raise issues on this GraphiQL [repository](https://github.com/graphql/graphiql) as we use it as a third party tool in Hasura console.
